MonoZelda/Scenes/DungeonScene.cs

```diff
@@ -41,15 +41,15 @@ public void LoadContent(ContentManager contentManager)
 
         //create player and player collision
         player = new Player();
-        Collidable playerHitbox = new Collidable(new Rectangle(100, 100, 50, 50), graphicsDevice, CollidableType.Player);
+        Collidable playerHitbox = new Collidable(new Rectangle(100, 100, 50, 50), CollidableType.Player);
```

Comment:
Should the player hitbox be created within player so it can be removed from game? Doesn't need to happen in this PR. Maybe make a ticket if you think it's something that should be done.

Reply:
Yes, creating player hitbox insided the player class would be ideal.